bin/fypp

```diff
@@ -154,6 +154,8 @@ _CLOSING_BRACKETS_FORTRAN = '})]'
 
 _ARGUMENT_SPLIT_CHAR_FORTRAN = ','
 
+_DEPFILE_QUOTE = re.compile(r'[$: ]')
```

Comment:
Do you have authoritative source describing the depfile format? According to what is written on the [relevant CMake manpage](https://cmake.org/cmake/help/latest/command/add_custom_command.html), ":" must not be escaped, but `$` (with `$$`), `#` (with `\#`) and `  `  with `\ `.

Reply:
You're right, Ninja is perfectly happy with using `$` to escape all of those, but make isn't. I'll change that.